targets/unity-v2/source/Shared/Internal/PlayFabHttp/PlayFabWWW.cs

```diff
@@ -55,30 +60,40 @@ private static IEnumerator SimpleCallCoroutine(string fullUrl, byte[] payload, A
             }
             else
             {
-                var putRequest = UnityWebRequest.Put(fullUrl, payload);
+                UnityWebRequest request;
+                if (method == "put")
+                {
+                    request = UnityWebRequest.Put(fullUrl, payload);
+                }
+                else
+                {
+                    var strPayload = System.Text.Encoding.UTF8.GetString(payload);
+                    request = UnityWebRequest.Post(fullUrl, strPayload);
```

Comment:
You are converting "GET" calls to "POST" which will make all GET calls fail.
(You're not handling get separately from post)